tests/integration_tests/partial_index/run.sh

```
@@ -0,0 +1,70 @@
+
+	# sync_diff can't check non-exist table, so we check expected tables are created in downstream first
+	check_table_exists partial_index.t1 ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists partial_index.t2 ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists partial_index.t3 ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+	check_table_exists partial_index.finish_mark ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
+
+	check_sync_diff $WORK_DIR $CUR/conf/diff_config.toml
+
+	cleanup_process $CDC_BINARY
+
+	run_sql "show create table partial_index.t1;" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} &&
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)